index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Document</title>
+        <link rel="stylesheet" type="text/css" href="style.css" />
+    </head>
+
+    <body class="container">
+        <header>
+            <div class="header">
+                <div class="logo">
+                   <a href=""><img src="My Logo.png" alt="" class="logo-img" /></a> 
+                </div>
+                <nav>
+                    <div class="menu">
+                        <img src="Union.svg" alt="">
+                    </div>
+                </nav>
+            </div>
+        </header>
+
+        <section>
+            <div class="container2">
+                <h1 class="first-div">I'm Alexander. Glad to see you</h1> 
+                <div class="second-div">
+                    <p>i'm a software developer! i can help you build a product, feature or
```

Comment:
[OPTIONAL] - Kindly adjust the section background images so that it covers the entire section to match the [templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).

Figma
![image](https://user-images.githubusercontent.com/78251927/235298752-146971f7-1fda-44e6-8cb2-b0a200e1ba22.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/235298224-dc20479f-7327-4b12-869b-64caadb3d5ea.png)